samples/Server/src/lib/identityClient.ts

```diff
@@ -8,15 +8,17 @@ import {
   TokenScope
 } from '@azure/communication-identity';
 import { CommunicationUserIdentifier } from '@azure/communication-common';
-import { getResourceConnectionString } from './envHelper';
+import { getResourceConnectionString, getAdminUserId } from './envHelper';
 
 // lazy init to allow mocks in test
 let identityClient: CommunicationIdentityClient | undefined = undefined;
 const getIdentityClient = (): CommunicationIdentityClient =>
   identityClient ?? (identityClient = new CommunicationIdentityClient(getResourceConnectionString()));
 
 // replicate here to allow for mocks in tests
-export const createUser = (): Promise<CommunicationUserIdentifier> => getIdentityClient().createUser();
+export const createUser = (): CommunicationUserIdentifier => {
```

Comment:
can you rename this to `createAdminUser` ? To avoid confusion with `createUserAndToken` below which creates a non-admin user 
```suggestion
export const createAdminUser = (): CommunicationUserIdentifier => {
```

Reply:
Actually `getAdminUser` would be more appropriate as it doesn't create a new admin

Reply:
I like getAdminUser thanks for the suggestion!